<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadData</class>
 <widget class="QMainWindow" name="DownloadData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Basic patient and visit data will always be included in the download. 
 
 Which additional data would you like to include in your CSV file? 
 
 Please note that adding more data will increase the complexity of the resulting CSV table.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="check_all_button">
        <property name="text">
         <string>Check all</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="uncheck_all_button">
        <property name="text">
         <string>Uncheck all</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="eckardt_scores">
        <property name="text">
         <string>Eckardt Score</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="gerd_scores">
        <property name="text">
         <string>GERD Score</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="barium_swallows">
        <property name="text">
         <string>TBE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="endoscopies">
        <property name="text">
         <string>EGD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="manometries">
        <property name="text">
         <string>Manometry</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="previous_therapies">
        <property name="text">
         <string>Previous Therapies</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="endoflips">
        <property name="text">
         <string>EndoFlip</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="medications">
        <property name="text">
         <string>Medication</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="botox_injections">
        <property name="text">
         <string>Botox Injection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="poems">
        <property name="text">
         <string>POEM</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="pneumatic_dilations">
        <property name="text">
         <string>Pneumatic Dilation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="lhms">
        <property name="text">
         <string>LHM</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="complications">
        <property name="text">
         <string>Complications</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="download_data_button">
        <property name="text">
         <string>Download data</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>check_all_button</tabstop>
  <tabstop>uncheck_all_button</tabstop>
  <tabstop>previous_therapies</tabstop>
  <tabstop>eckardt_scores</tabstop>
  <tabstop>gerd_scores</tabstop>
  <tabstop>medications</tabstop>
  <tabstop>manometries</tabstop>
  <tabstop>barium_swallows</tabstop>
  <tabstop>endoscopies</tabstop>
  <tabstop>endoflips</tabstop>
  <tabstop>botox_injections</tabstop>
  <tabstop>poems</tabstop>
  <tabstop>pneumatic_dilations</tabstop>
  <tabstop>lhms</tabstop>
  <tabstop>complications</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
